content/en/apps/tutorials/cht-academy.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Overview of CHT Academy"
+linkTitle: CHT Academy
+weight: 5
```

Comment:
Please make sure this shows up first within the Tutorials section. I think it makes sense for people to see this before other tutorial content.